### PR TITLE
[ZT] add mailto feature to block page, general cleanup

### PR DESCRIPTION
**Pull request by ranbel**

### CODE REVIEW

File: content/cloudflare-one/policies/filtering/configuring-block-page.md [hunk 1]
```diff
@@ -4,109 +4,77 @@ title: Block page
...
+You can add a Mailto link to your custom block page, which allows users to directly email you about the blocked site. When users click **Contact your Administrator** on your block page, an email template opens with the email address and subject line you configure, as well as the following diagnostic information:
 
-![Final block page](/cloudflare-one/static/documentation/policies/final-block-page.png)
+| Field        | Description                                |
+| ------------- | ------------------------------------------- |
+| Site URL     | The URL of the blocked page.   |
+| Rule ID      | The ID of the Gateway policy that blocked the page. |
+| Source IP    | The public source IP of the user device. |
+| Account ID   | The Cloudflare account associated with the block policy. |
+| User ID      | The ID of the user who visited the page. This is generated by the WARP client. |
```
Comment (ranbel) [inline content/cloudflare-one/policies/filtering/configuring-block-page.md]: @Encore-Encore 
How do I identify the user from the "User ID"? The User ID does not show up when I click a user in **My Team** > **Users**.

Comment (ranbel) [reply content/cloudflare-one/policies/filtering/configuring-block-page.md]: Currently User ID can only be obtained from the API.  (uid field)
https://api.cloudflare.com/#zero-trust-users-get-users

File: content/cloudflare-one/policies/filtering/configuring-block-page.md [hunk 2]
```diff
@@ -4,109 +4,77 @@ title: Block page
...
+Users will now see a custom message when they are blocked by this HTTP policy.
 
-![Enable block page](/cloudflare-one/static/documentation/policies/block-page-toggle.png)
+## Enable the block page for DNS policies
 
-1.  Click **Save**.
+1. On the [Zero Trust dashboard](https://dash.teams.cloudflare.com), navigate to **Policies** > **DNS**.
+2. Find the policy for which you would like to display a block page and click **Edit**. You can only enable the block page for policies with a Block action.
+3. Scroll down to the **Configure policy settings** step.
+4. Toggle the **Display block page** switch.
```
Comment (crwaters16) [inline content/cloudflare-one/policies/filtering/configuring-block-page.md]: Later on you use "toggle on" vs just "toggle" - I don't think we have guidance on how to use toggle as a verb, so I'd recommend picking one for consistency

**Review by crwaters16 — APPROVED** [review]
One small comment, otherwise looks good

File: content/cloudflare-one/policies/filtering/configuring-block-page.md [hunk 3]
```diff
@@ -4,109 +4,77 @@ title: Block page
...
-- Adding a global block message
-- Choosing a background color
+You can customize the block page by making global changes that will show up every time a user navigates to a block page, independently of the type of rule (DNS or HTTP) that is blocking the website. 
 
-All of these elements will display in addition to two fields specific to the policy that's blocking the website:
+To apply customizations to your block page:
 
-- **The policy-specific block message** you've specified while creating the policy. This message is displayed under your global block message, and above the Rule ID.
+1. On the [Zero Trust dashboard](https://dash.teams.cloudflare.com), navigate to **Settings** > **General**.
+2. Under **Block page**, toggle on the custom block page feature.
```
Comment (ranbel) [inline content/cloudflare-one/policies/filtering/configuring-block-page.md]:
```suggestion
2. Under **Block page**, enable the custom block page feature.
```

File: content/cloudflare-one/policies/filtering/configuring-block-page.md (hunk 2 above)
Comment (ranbel) [inline content/cloudflare-one/policies/filtering/configuring-block-page.md]:
```suggestion
4. Enable **Display block page**.
```

**Pull request merged**